poi/src/main/java/org/apache/poi/hssf/usermodel/HSSFWorkbook.java

```diff
@@ -31,17 +31,8 @@ Licensed to the Apache Software Foundation (ASF) under one or more
 import java.io.PrintWriter;
 import java.nio.charset.Charset;
 import java.security.GeneralSecurityException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.NoSuchElementException;
-import java.util.Set;
+import java.util.*;
```

Comment:
can you keep the explicit imports and remove the * import?

Reply:
Sorry, this was automatically formatted by my IDE, will remove it

Reply:
Done